Recrutify/Recrutify.Host/Controllers/SchedulesController.cs

```diff
@@ -22,9 +22,15 @@ public SchedulesController(IScheduleService scheduleService)
         }
 
         [HttpGet]
-        public Task<IEnumerable<ScheduleDTO>> GetByUserPrimarySkillAsync([FromQuery, Required] Guid projectId, [FromQuery, Required] DateTime date, [FromQuery, Required] Guid primarySkillId)
+        public Task<IEnumerable<ScheduleDTO>> GetByUserPrimarySkillAsync([FromQuery, Required] Guid projectId, [FromQuery] DateTime? date, [FromQuery, Required] Guid primarySkillId)
         {
             return _scheduleService.GetByUserPrimarySkillAsync(projectId, date, primarySkillId);
         }
+
+        [HttpGet("date")]
```

Comment:
current_user think will be better, cause we have date in both endpoints

Reply:
+